pipeline.ts

```diff
@@ -48,13 +45,14 @@ export function createRedisPipeline(
   }
 
   async function send(cmds: string[]): Promise<RawReplyOrError[]> {
+    const writerAsBuffer = connection.writer! as BufWriter;
```

Comment:
I think this type assertion is unnecessary.
```suggestion
    const writerAsBuffer = connection.writer!;
```